dbms/src/Common/tests/gtest_sensitive_data_masker.cpp

```diff
@@ -0,0 +1,220 @@
+#include <Common/Exception.h>
+#include <Common/SensitiveDataMasker.h>
+#include <Poco/AutoPtr.h>
+#include <Poco/Util/XMLConfiguration.h>
+#include <Poco/XML/XMLException.h>
+
+#pragma GCC diagnostic ignored "-Wsign-compare"
+#ifdef __clang__
+#    pragma clang diagnostic ignored "-Wzero-as-null-pointer-constant"
+#    pragma clang diagnostic ignored "-Wundef"
+#endif
```

Comment:
Copy & paste from another test, may be not needed here.

Reply:
It is needed for gtest itself.